panedrlite/panedrlite/panedr.py

```diff
@@ -395,14 +396,14 @@ def edr_strings(data, file_version, n):
 
 def is_frame_magic(data):
     """Unpacks an int and checks whether it matches the EDR frame magic number
-    
+
     Does not roll the reading position back.
     """
     magic = data.unpack_int()
     return magic == -7777777
 
 
-def edr_to_df(path, verbose=False):
+def read_edr(path, verbose=False):
```

Comment:
Does this PR supersede #33 or is it meant to follow it? (i.e. which should get merged first)

Reply:
Oh, I didn't mean to create this branch here from my branch for #33... Too new to using git still. #33 should probably be merged first, or at least the changes approved. Alternatively, I could replace this version of panedr.py with the one from the master branch (as I originally intended)